qa/tasks/cephfs/fuse_mount.py

```diff
@@ -57,16 +57,12 @@ def _mount(self, mntopts, check_status):
            self.client_config.get('valgrind') is not None:
             daemon_signal = 'term'
 
+        self.client_remote.run(args=f'mkdir -p -v {self.hostfs_mntpt}',
+                               timeout=60, stderr=StringIO())
```

Comment:
Is there a reason to do `stderr=StringIO()` if we're not going to look at it?

Reply:
Yep, that can be removed as well!

Removed it in the current push! Thanks!